lms/djangoapps/instructor/views/api.py

```diff
@@ -2642,3 +2642,22 @@ def mark_student_can_skip_entrance_exam(request, course_id):  # pylint: disable=
         'message': message,
     }
     return JsonResponse(response_payload)
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_global_staff
+@require_POST
+def start_certificate_generation(request, course_id):
+    """
+    Start generating certificates for all students enrolled in given course.
+    """
+    course_key = CourseKey.from_string(course_id)
+    task = instructor_task.api.generate_certificates_for_all_students(request, course_key)
+    message = _('Certificate generation task for all students of this course has been started. '
+                'You can view the status of the generation task in the "Pending Instructor Tasks" section.')
```

Comment:
@mattdrayer Sorry, a late-breaking change. A Doc backlog story recently changed the UI so that it is now "Pending Tasks" section rather than "Pending Instructor Tasks." 
Please remove the word "Instructor".